### DISCUSSION TITLE
Upstream Rust language support

### DISCUSSION THREAD

**Discussion by madsmtm** [corrosion-rs/corrosion#471] [General]
Hey, I've opened [this issue upstream](https://gitlab.kitware.com/cmake/cmake/-/issues/25492) to track support for built-in Rust language support, I thought you might find that useful and/or might have comments about how it relates to this project's needs.

**Comment by jschwe** [comment]
I've actually loosely been following the previous discussion on the CMake discord, but it doesn't seem like this is going to go anywhere (at least in the near to medium term). In my opinion, adding proper upstream support for Rust (i.e. not just run `cargo build` and massaging the artifacts) does not fit well with CMake at all.
